src/main/scala/mesosphere/marathon/api/akkahttp/v2/AppsController.scala

```diff
@@ -136,6 +159,53 @@ class AppsController(
 
   private def patchSingle(appId: PathId)(implicit identity: Identity): Route = ???
 
+  private[this] def updateMultiple(partialUpdate: Boolean, allowCreation: Boolean)(implicit identity: Identity): Route = {
```

Comment:
As we've discussed the identity probably should not be passed implicitly but rather with the request object. Aside from that this seems to be a helper method so why not pass the identity explicitly?